pkg/ocihook/ocihook.go

```diff
@@ -321,6 +326,22 @@ func getPortMapOpts(opts *handlerOpts) ([]gocni.NamespaceOpts, error) {
 	return nil, nil
 }
 
+func getIPAddressOpts(opts *handlerOpts) ([]gocni.NamespaceOpts, error) {
+	if opts.containerIP != "" {
+		if rootlessutil.IsRootlessChild() {
+			return nil, fmt.Errorf("containerIP assignment is not supported in rootless mode")
```

Comment:
Can be another PR, but you can implement this for rootless mode too.
Not reachable from the host though

Reply:
I think I can make rootless support in single PR